pkg/virt-controller/watch/util/util.go

```diff
@@ -112,3 +114,39 @@ func CreateDataVolumeManifest(clientset kubecli.KubevirtClient, dataVolumeTempla
 
 	return newDataVolume, nil
 }
+
+// NodeSelectorRequirementsAsSelector converts the []NodeSelectorRequirement api type into a struct that implements
+// labels.Selector.
+func NodeSelectorRequirementsAsSelector(nsm *[]corev1.NodeSelectorRequirement) (labels.Selector, error) {
```

Comment:
You can keep it closer to where is is being used. Keeping this in `util.go` is an anti-pattern.